### PR TITLE
[Kotlin migration] FieldEditLineTest.kt

### PR DESCRIPTION
**Pull request by Prince-kushwaha**
## Purpose / Description
Kotlin migration of FieldEditLineTest
- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/androidTest/java/com/ichi2/anki/FieldEditLineTest.kt [hunk 1]
```diff
@@ -13,63 +13,61 @@
...
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.ichi2.utils.KotlinCleanup
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.`is`
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.concurrent.atomic.AtomicReference
+
+@KotlinCleanup("is -> equalTo")
+@RunWith(AndroidJUnit4::class)
```
Comment (david-allison) [inline AnkiDroid/src/androidTest/java/com/ichi2/anki/FieldEditLineTest.kt:27]: I'd add a cleanup to use scope functions as well

File: AnkiDroid/src/androidTest/java/com/ichi2/anki/FieldEditLineTest.kt [hunk 2]
```diff
@@ -13,63 +13,61 @@
...
     }
 
-
-    @NonNull
-    protected FieldEditLine getFieldEditLine() {
-        AtomicReference<FieldEditLine> l = new AtomicReference<>();
-        mActivityRule.getScenario().onActivity(a -> l.set(new FieldEditLine(a)));
-        return l.get();
-    }
+    protected val fieldEditLine: FieldEditLine
```
Comment (david-allison) [inline AnkiDroid/src/androidTest/java/com/ichi2/anki/FieldEditLineTest.kt:61]: cleanup to move this back to a method

**Review by david-allison — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/androidTest/java/com/ichi2/anki/FieldEditLineTest.kt (hunk 1 above)
Comment (Arthur-Milchior) [reply AnkiDroid/src/androidTest/java/com/ichi2/anki/FieldEditLineTest.kt:27]: @Prince-kushwaha 
Do you plan to do it?
No emergency, I just want to know whether this is an abandonned commit or not

**Review by david-allison — APPROVED** [review]

**Review by mikehardy — APPROVED** [review]
LGTM thanks!

**Pull request merged**